src/ui/public/config/defaults.js

```diff
@@ -109,7 +109,7 @@ define(function (require) {
       'visualization:colorMapping': {
         type: 'json',
         value: JSON.stringify({
-          'Count': '#57c17b'
+          'Count': '#6eadc1'
```

Comment:
Do we have new seed colors to replace them with? I wasn't under the impression we did?